.github/workflows/visual-regression-tests.yml

```
@@ -0,0 +1,31 @@
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  visual-regression-test-components:
```

Just a rename, since we're not looking at components here:

```suggestion
  visual-regression-test:
```